assemblyline_core/dispatching/dispatch_hash.py

```diff
@@ -149,13 +149,13 @@ def all_dispatches(self) -> Dict[str, Dict[str, bytes]]:
         """Load the entire table of things that should currently be running."""
         rows = retry_call(self.client.hgetall, self._dispatch_key)
         output = {}
-        for key, key in rows.items():
```

Comment:
this hurts me

Reply:
I've been trying to figure out what was wrong with that code for a week just staring at it.